pkg/controllers/cassandra/nodepool/nodepool.go

```diff
@@ -78,22 +78,32 @@ func (e *defaultCassandraClusterNodepoolControl) removeUnusedStatefulSets(
 
 func (e *defaultCassandraClusterNodepoolControl) labelSeedNodes(
 	cluster *v1alpha1.CassandraCluster,
+	nodePool *v1alpha1.CassandraClusterNodePool,
 	set *appsv1beta1.StatefulSet,
 ) error {
-	// TODO: make number of seed nodes configurable
-	pod, err := e.pods.Pods(cluster.Namespace).Get(fmt.Sprintf("%s-%d", set.Name, 0))
-	if err != nil {
-		glog.Warningf("Couldn't get stateful set pod: %v", err)
-		return nil
-	}
-
-	// only label if the current label is incorrect
-	if pod.Labels["seed"] != "true" {
-		podCopy := pod.DeepCopy()
-		podCopy.Labels["seed"] = "true"
-		_, err := e.kubeClient.CoreV1().Pods(podCopy.Namespace).Update(podCopy)
+	for i := int64(0); i < nodePool.Replicas; i++ {
+		pod, err := e.pods.Pods(cluster.Namespace).Get(fmt.Sprintf("%s-%d", set.Name, i))
 		if err != nil {
-			return err
+			glog.Warningf("Couldn't get stateful set pod: %v", err)
+			return nil
+		}
+
+		// default to not a seed
+		desiredLabel := "false"
+
+		// label first n as seeds
+		if i < nodePool.Seeds {
+			desiredLabel = "true"
+		}
+
+		// only label if the current label is incorrect
+		if pod.Labels["seed"] != desiredLabel {
```

Comment:
yep, not something created by this PR, but `seed` isn't an acceptable label imo. difficult for a user to understand why it is there, and could be removed by accident.